tests/test_runlib.py

```diff
@@ -726,6 +756,13 @@ def test_normalize_line_endings(self):
       for path in paths:
         os.remove(path)
 
+  def test_wrong_metadata_directory(self):
+    """Test record stop with wrong metadata directory"""
+    with self.assertRaises(IOError):
```

Comment:
Remind me, this is dependent on the python version, right? If so, we should add one with the "other" exception as well.